tests/lib/devicetree/api/src/main.c

```diff
@@ -1376,8 +1404,17 @@ static void test_devices(void)
 	 */
 	val = 0;
 #define INC(inst_ignored) do { val++; } while (0);
-	DT_INST_FOREACH_STATUS_OKAY(INC)
+	DT_INST_FOREACH_STATUS_OKAY(INC);
```

Comment:
Why? This change doesn't seem related to this PR and it will lead to an extraneous semicolon in the generated output. I'd be nervous some linter somewhere will complain.